Language/sv-SE/Session.php

```diff
@@ -14,6 +14,6 @@
     'invalidSavePath'        => 'Session: Angivna sökvägen "{0}" är inte en katalog, existerar inte eller kan inte skapas.',
     'writeProtectedSavePath' => 'Session: Angivna sökvägen "{0}" är inte skrivbar för PHP processen.',
     'emptySavePath'          => 'Session: Ingen sökväg har angetts.',
-    'invalidSavePathFormat'  => 'Session: Ogiltigt format för Redis: {0}',
-    'invalidSameSiteSetting' => 'Session: SameSite måste vara None, Lax, Strict, eller en blank sträng. Givet: {0}',
+    'invalidSavePathFormat'  => 'Session: Ogiltig sökväg för Redis: "{0}"',
```

Comment:
Was this a mistake before?

Reply:
@tangix you can open a new PR if that would be faster for you.